examples/sgemm/SGEMM_kernels.ispc

```diff
@@ -85,8 +94,9 @@ static task void SGEMM_naive_task(uniform float matrixA[], uniform float matrixB
 
 export void SGEMM_naive_withTasks(uniform float matA[], uniform float matB[], uniform float matC[],
                                     uniform int M, uniform int N, uniform int K) {
-    // Generate more tasks than threads, to enable some load balancing:
-    uniform int numTasks = num_cores() * 4;
+    // Task must have at least programCount columns to process, but we require the matrix
+    // K size to be a miltiple of programCount, so it's safe to divide here
+    uniform int numTasks = M / programCount;
```

Comment:
Either comment or code is wrong. Comment talks about K, code uses M.

Reply:
Ctrl-C, Ctrl-V error, fixed.

Reply:
Ctrl-C, Ctrl-V error, fixed.